Makefile

```
@@ -1,27 +1,32 @@
 SENTRY_PATH := `python -c 'import sentry; print sentry.__file__.rsplit("/", 3)[0]'`
 UNAME_S := $(shell uname -s)
 
-develop: setup-git install-yarn
-	pip install "pip>=9,<10"
-	pip install -e git+https://github.com/getsentry/sentry.git#egg=sentry[dev,optional]
-	pip install -e ".[tests]"
+PIP_VERSION := 19.1.1
+PIP_OPTS := --no-use-pep517 --disable-pip-version-check
+
+develop: ensure-pip setup-git install-yarn
+	pip install -e git+https://github.com/getsentry/sentry.git#egg=sentry[dev,optional] $(PIP_OPTS)
+	pip install -e ".[tests]" $(PIP_OPTS)
 	yarn install
 
+ensure-pip:
+	pip install "pip==$(PIP_VERSION)"
+
 install-yarn:
 	@echo "--> Installing Node dependencies"
 	@hash yarn 2> /dev/null || npm install -g yarn
 	# Use NODE_ENV=development so that yarn installs both dependencies + devDependencies
 	NODE_ENV=development yarn install --ignore-optional
 
 install-tests: develop
-	pip install .[tests]
+	pip install ".[tests]" $(PIP_OPTS)
```

```suggestion
	pip install $(PIP_OPTS) ".[tests]" 
```
Putting options as early as possible is better to resolve the ambiguity. We may even pull them before the `install` command if supported to prevent any "accidental" package names sneaking into these options.

---

I agree. The only reason this is here is because this is just how it is currently done in getsentry/sentry.

---

Consistency for consistency's sake, especially for poor practices is not a good thing. Since this is not very important and dangerous, I think it can stay that way but I'd just beware of this consistency trap.